app/assets/stylesheets/application/contributors.scss

```diff
@@ -1,9 +1,49 @@
-.hideContent {
-  display: none;
-}
+.contributorModalBody {
+	display: flex;
+	flex-direction: row;
+
+	.contributorModalBodyPhoto {
+    flex: 0 1 30%;
+    @media screen and (max-width: $small) {
+      flex: 0 1 25%;
+      .contributorModalAvatarMedium img.Avatar__image___2cXeU {
+        width: 100px;
+        height: 100px;
+      }
+    }
+  }
+
+	.contributorModalBodyCopy {
+    margin-left: $size-18;
```

Comment:
I think this can be removed so we can define spacing around the modal content in the parent box element.

Reply:
I've left this in for now. Without it, the content is crammed up against the avatar at larger breakpoints. Not sure what you meant by defining spacing around the modal content in the parent box element.
![nospacing](https://user-images.githubusercontent.com/16218249/48752521-ed6cda80-ec46-11e8-8a96-1714e80f240a.png)